streaming/base/dataset.py

```diff
@@ -752,6 +752,7 @@ def __iter__(self) -> Iterator[Dict[str, Any]]:
         """
         # Lazily create the worker barrier's FileLock, which contains a threading Lock, which is
         # unpickleable.
+
```

Comment:
is this a spurious newline

Reply:
lint is a cultprit ;). Removed it in https://github.com/mosaicml/streaming/pull/171/commits/5b5c10450637d4ccdbc8b5c2403a7d18f89b169b.